libs/jsonforms-components/src/lib/Controls/Inputs/InputTextControl.tsx

```diff
@@ -20,6 +20,10 @@ export const GoAInputText = (props: GoAInputTextProps): JSX.Element => {
     uischema?.options?.componentProps?.autoCapitalize === true || uischema?.options?.autoCapitalize === true;
   const readOnly = uischema?.options?.componentProps?.readOnly ?? false;
 
+  if (uischema?.options?.isStepperReview) {
+    return <div data-testid="input-text-control-review">{data}</div>;
```

Comment:
@solittlework can we refactor this section of code to check if isStepperReview is set and rendering the data.  It looks like some of these controls are the same?

Reply:
Yeah, I found the simple one is good for most of the cases. But, for some cases, I expect we need to create the text based on the options. This code just provide a framework or placeholder for the future work.